.github/workflows/CI.yml

```diff
@@ -1019,9 +1019,23 @@ jobs:
           msystem: ucrt64
           update: true
           install: >-
+            git
+            mingw-w64-ucrt-x86_64-cmake
+            mingw-w64-ucrt-x86_64-cppwinrt
+            mingw-w64-ucrt-x86_64-curl
```

Comment:
winssl variant replaces (conflicts with) the original version so this line should be removed

Reply:
Oh, got it. I think I misunderstood.